core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -808,7 +808,6 @@ private[spark] class TaskSetManager(
         if (tasksSuccessful == numTasks) {
           isZombie = true
         }
-        maybeFinishTaskSet()
```

Comment:
why you remove this ?

Reply:
Before we were going to merge this PR, the redundant call to `maybeFinishTaskSet()` was told to be removed in an earlier comment in this PR.